tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpClientTestSupport.java

```diff
@@ -155,6 +155,7 @@ protected ActiveMQServer createServer(int port) throws Exception {
       server.getConfiguration().setPagingDirectory(server.getConfiguration().getPagingDirectory() + port);
       server.getConfiguration().setJMXManagementEnabled(true);
       server.getConfiguration().setMessageExpiryScanPeriod(5000);
+      server.getConfiguration().setUseLegacyAMQPQueueNaming(false);
```

Comment:
@gemmellr i really/ideally like to have the tests run with it both on and off, any idea how to do that without a massive copy paste exercise.